### PR TITLE
Use the new riff-raff.yaml configuration format

### PR DESCRIPTION
**Pull request by rtyley**
## Why are you doing this?

Because of the [stick](https://github.com/guardian/riff-raff/pull/435), not the :carrot: :

image

Done after looking at:

* https://github.com/guardian/support-frontend/pull/1
* https://github.com/guardian/bard/pull/35
* https://github.com/guardian/media-atom-maker/pull/218
* https://github.com/guardian/ophan/pull/2051

The yaml verifies in https://riffraff.gutools.co.uk/configuration/validation

### CODE REVIEW

File: frontend/conf/riff-raff.yaml [hunk 1]
```diff
@@ -8,6 +8,9 @@ deployments:
       amiTags:
         Recipe: xenial-membership
         AmigoStage: PROD
+      cloudFormationStackName: "Memb"
+      prependStackToCloudFormationStackName : false
+      appendStageToCloudFormationStackName : true
```
Comment (rtyley) [inline frontend/conf/riff-raff.yaml]: Without these three lines, we get:

image

File: frontend/conf/riff-raff.yaml [hunk 2]
```diff
@@ -8,9 +8,6 @@ deployments:
       amiTags:
         Recipe: xenial-membership
         AmigoStage: PROD
-      cloudFormationStackName: "Memb"
-      prependStackToCloudFormationStackName : false
-      appendStageToCloudFormationStackName : true
```
Comment (rtyley) [inline frontend/conf/riff-raff.yaml]: As @davidfurey pointed out, adding this tagging in the AWS console is enough for Riff Raff to find the `Memb-PROD` stack:

image

**Comment by rtyley** [comment]
As of 019a9aa riffraff is happy to deploy the build: https://riffraff.gutools.co.uk/deployment/view/67eb8f1a-03b8-4c35-a600-e3acaa70cf0d

image

...though due to our current stability issues it would not surprise me if some of those boxes fail to come up (you can see that 1 has, at least)

**Comment by rtyley** [comment]
Merging, as I got a verbal approval from @davidfurey so long as I did the above changes :)

**Comment by davidfurey** [comment]
👍

**Comment by rtyley** [comment]
This manual deploy of `master` eventually [did](https://riffraff.gutools.co.uk/deployment/view/48cf68d3-901d-4d87-9a39-b8c060a7230a) succeed:

image

**Pull request merged**